src/js/facility-locator/utils/helpers.js

```diff
@@ -0,0 +1,76 @@
+export const keyMap = {
+  TAB: 9,
+  ENTER: 13,
+  ESCAPE: 27,
+  SPACE: 32,
+  UP: 38,
+  DOWN: 40
+};
+
+export function getDirection(code) {
+  if (code === keyMap.UP) return -1;
+  if (code === keyMap.DOWN) return 1;
+  return false;
+}
+
+export function getSelection(options, queryType) {
+  let selection = options[0];
+  if (queryType) {
+    const selectionType = queryType;
+    selection = options.reduce((acc, elem) => {
+      if (elem.id && elem.id === selectionType) acc = elem; // eslint-disable-line no-param-reassign
+      return acc;
+    }, selection);
+  }
+  return { selection, id: options.indexOf(selection) };
+}
+
+export function isSelect(code) {
+  return code === keyMap.ENTER || code === keyMap.SPACE;
+}
+
+export function isTraverse(keyEvent) {
+  const { keyCode: code } = keyEvent;
+  const result = code === keyMap.UP || code === keyMap.DOWN;
+  if (result) keyEvent.preventDefault();
```

Comment:
This feels odd to have in an `isTraverse` function. Reading that name, I wouldn't expect it to call anything on the event itself. Applies to a couple other of these functions, too.

Reply:
Good point, I will relocate the preventDefault.